torchaudio/transforms.py

```diff
@@ -284,22 +291,102 @@ class SpectogramToDB(object):
     """
     def __init__(self, stype="power", top_db=None):
         self.stype = stype
-        if top_db is not None and top_db > 0:
-            top_db = -top_db
```

Comment:
You're now assuming `top_db` is positive, as in librosa. The docstring needs to change accordingly, and you will want to throw an error for negative values like librosa.